app/process/watcher/consensus-message/consensus-message.go

```diff
@@ -1,3 +1,48 @@
-package consensus_message
+package consensusmessage
 
-// todo consensus message watcher
+import (
+	hederasdk "github.com/hashgraph/hedera-sdk-go"
+	"github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera"
+	"github.com/limechain/hedera-eth-bridge-validator/app/process/watcher/publisher"
+	"github.com/limechain/hedera-watcher-sdk/queue"
+	"log"
```

Comment:
Not for this PR but in general think about including a more powerful logging library that will allow you to do stuff like `log.Infof`, `log.Warnf` and indicate error levels. It will also allow you to disable certain level messages